infrastructure/modules/databases/variables.tf

```
@@ -0,0 +1,6 @@
+variable "environment" {
+  description = "Namespace for all resources. Eg 'production' 'dev-jane'"
+}
+variable "postgresql_server_name" {
+  description = "Admin password for the postgresql server"
```

I think this description is not correct. Maybe something like this?

```suggestion
  description = "Name of the PostgreSQL server, where databases and users should be created"
```